site/content/kapp/docs/develop/config.md

```diff
@@ -150,6 +150,30 @@ waitRules:
   resourceMatchers:
   - apiVersionKindMatcher: {apiVersion: corp.com/v1, kind: Application}
 ```
+#### waitRules for Custom Resources
```

Comment:
waitRules in general can be used for all kinds of resources, including custom resources, so I think we can continue under `waitRules` and remove this heading.

Reply:
+1